x/nft/go.mod

```diff
@@ -153,6 +153,4 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-// Fix upstream GHSA-h395-qcrw-5vmq and GHSA-3vp4-m3rf-835h vulnerabilities.
-// TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
-replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.9.1
+replace github.com/cosmos/cosmos-sdk => ../../.
```

Comment:
These imperatively must be removed from here, and simapp must use pseudo version.